drivers/SmartThings/zigbee-lock/src/init.lua

```diff
@@ -291,6 +293,42 @@ local name_slot = function(driver, device, command)
   end
 end
 
+local migrate = function(driver, device, command)
+  local lock_users = {}
+  local lock_credentials = {}
+  local lock_codes = lock_utils.get_lock_codes(device)
+  local ordered_codes = {}
+
+  for code in pairs(lock_codes) do
+      table.insert(ordered_codes, code)
+  end
+
+  table.sort(ordered_codes)
+  for index = 1, #ordered_codes do
```

Comment:
I don't think this will work on a real device. The code indices aren't guaranteed to be contiguous.